bower.json

```diff
@@ -9,16 +9,38 @@
     "package.json"
   ],
   "dependencies": {
-    "polymer": "^1.9.1",
-    "d2l-colors": "^2.4.0",
-    "d2l-icons": "^3.3.0",
-    "iron-resizable-behavior": "^1.0.6",
+    "polymer": "1.9 - 2",
+    "d2l-colors": "^2.4.0 || ^3.1.0",
+    "d2l-icons": "^4.4.0",
+    "iron-resizable-behavior": "1 - 2",
     "Stickyfill": "^1.1.4"
   },
   "devDependencies": {
-    "d2l-demo-template": "^0.0.12",
-    "iron-component-page": "^1.1.9",
+    "d2l-demo-template": "0.0.12 - 1",
```

Comment:
I've been ditching the demo template and switching to use Polymer's demo stuff as it plays much nicer when the element gets added to webcomponents.org.